test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/Tests/BulkUpdateE2ETests.cs

```diff
@@ -0,0 +1,456 @@
+﻿//---------------------------------------------------------------------

```

Comment:
I think you should add tests that verify the following scenarios:
- entities and related entities that are not new but have received some changes are properly handled and deserialized
- If the bulk modified entities, we should be able to see that the entities in the entity tracker have also been updated
- nested entities with property changes beyond the ID